feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/bindingadapter/TextViewBinding.kt

```diff
@@ -0,0 +1,69 @@
+package io.github.droidkaigi.confsched2019.session.ui.bindingadapter
+
+import android.graphics.Bitmap
+import android.graphics.drawable.BitmapDrawable
+import android.graphics.drawable.Drawable
+import android.widget.TextView
+import androidx.core.graphics.drawable.DrawableCompat
+import com.squareup.picasso.Picasso
+import com.squareup.picasso.Target
+import jp.wasabeef.picasso.transformations.CropCircleTransformation
+
+fun loadImage(
```

Comment:
I don't want this file in bindingadapter package. Because of bindingadapter means DataBinding Binding Adapter. 🙇 
You can write this function in SpeechSessionItem.kt and you can omit nullable codes 👀

Reply:
Ok, I'll move it 👍